src/pipecat/processors/frame_processor.py

```diff
@@ -330,12 +327,10 @@ async def __cancel_push_task(self):
         await self.__push_frame_task
 
     async def __push_frame_task_handler(self):
-        running = True
-        while running:
+        while True:
             try:
                 (frame, direction) = await self.__push_queue.get()
                 await self.__internal_push_frame(frame, direction)
-                running = not isinstance(frame, EndFrame)
```

Comment:
This is to allow any `FrameProcessor` to still push frames after an `EndFrame`. This is useful for testing purposes. In practice this shouldn't really happen.